uicomponent-compose/other/build.gradle

```diff
@@ -10,6 +11,14 @@ plugins {
 apply from: rootProject.file("gradle/android.gradle")
 
 android {
+    buildTypes {
+        debug {
+            resValue "string", "app_version_name", "$AppVersions.versionName-debug"
+        }
+        release {
```

Comment:
Thank you for tackling the issue that how to refer to the application version name from the library module 👍 

In my opinion, I think we can just define `resValue "string", "app_version_name", AppVersions.versionName` at `defaultConfig`. I think it is enough for this case.

Because `debug-buildType` at library module doesn't always mean `debug-buildType` at app module.
If app module defined other buildTypes and fall back to `debug-buildType` as library modules,  the problem will occur.

How do you think about it? 👀

Reply:
> Because debug-buildType at library module doesn't always mean debug-buildType at app module.
If app module defined other buildTypes and fall back to debug-buildType on library modules, the problem will occur.

That's true.
Thank you for your advice, looks good.
I'll fix it!

Reply:
I have fixed it, so please review it again:pray: